macOS/UnitTests/Subscription/Mocks/MockSubscriptionManagerV2.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  MockSubscriptionManagerV2.swift
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+@testable import Subscription
+import JWTKit
+@testable import Networking
+import Common
+
+class MockSubscriptionManagerV2: SubscriptionManagerV2 {
```

Comment:
This is a duplicate, Subscription mocks are already available in `SubscriptionTestingUtilities/Managers`, please use that one and feel free to improve them if required